diff --git a/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/Content/index.js b/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/Content/index.js
index c8e1e3f..d5fa905 100644
--- a/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/Content/index.js
+++ b/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/Content/index.js
@@ -69,14 +69,14 @@ function H({
             titleImages: o
           }
         ) }),
-        !a && /* @__PURE__ */ e.jsx(d, { items: f }),
-        !u && !t && !a && j,
+        /* @__PURE__ */ e.jsx(d, { items: f }),
+        !u && !t && j,
         !t && m && i && /* @__PURE__ */ e.jsx(c, { className: n.colorModeToggle })
       ] }),
       right: /* @__PURE__ */ e.jsxs(e.Fragment, { children: [
         (a || !t && !m) && i && /* @__PURE__ */ e.jsx(c, { className: n.colorModeToggle }),
         /* @__PURE__ */ e.jsx(d, { items: p }),
-        !v.disabled && !a && /* @__PURE__ */ e.jsx(L, {})
+        !v.disabled && /* @__PURE__ */ e.jsx(L, {})
       ] })
     }
   );
diff --git a/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/Layout/index.js b/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/Layout/index.js
index 7611bf4..532f373 100644
--- a/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/Layout/index.js
+++ b/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/Layout/index.js
@@ -39,8 +39,8 @@ function M({
       }),
       className: s(
         "navbar",
-        !o && "navbar--fixed-top",
-        o && n.navbarLanding,
+        // !o && "navbar--fixed-top",
+        n.navbarLanding,
         e && [
           n.navbarHideable,
           !l && n.navbarHidden
