diff --git a/node_modules/@swmansion/t-rex-ui/dist/assets/index6.css b/node_modules/@swmansion/t-rex-ui/dist/assets/index6.css
index dae2c68..15638e5 100644
--- a/node_modules/@swmansion/t-rex-ui/dist/assets/index6.css
+++ b/node_modules/@swmansion/t-rex-ui/dist/assets/index6.css
@@ -1 +1 @@
-._navbarHideable_2jugh_1{transition:transform var(--ifm-transition-fast) ease}._navbarHidden_2jugh_5{transform:translate3d(0,calc(-100% - 2px),0)}[class*=navbar-sidebar__backdrop]{background-color:var(--swm-sidebar-backdrop)!important}._navbarLanding_2jugh_13{position:relative;background-color:transparent}
+._navbarHideable_2jugh_1{transition:transform var(--ifm-transition-fast) ease}._navbarHidden_2jugh_5{transform:translate3d(0,calc(-100% - 2px),0)}[class*=navbar-sidebar__backdrop]{background-color:var(--swm-sidebar-backdrop)!important}._navbarLanding_2jugh_13{position:fixed;width:100%;background-color:var(--swm-landing-background);transform:none;}
diff --git a/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/Content/index.js b/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/Content/index.js
index c8e1e3f..5050e8f 100644
--- a/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/Content/index.js
+++ b/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/Content/index.js
@@ -54,9 +54,10 @@ function H({
   titleImages: o,
   isAlgoliaActive: s,
   useLandingLogoDualVariant: b,
-  isThemeSwitcherShown: i
+  isThemeSwitcherShown: i,
+  landingItems: lx = []
 }) {
-  const t = h() === "mobile", { isDocumentation: m, isLanding: a } = g(), v = I(), l = z(), [f, p] = y(l), u = l.find((x) => x.type === "search"), j = s ? /* @__PURE__ */ e.jsx(S, {}) : /* @__PURE__ */ e.jsx("div", {});
+  const t = h() === "mobile", { isDocumentation: m, isLanding: a } = g(), v = I(), l = z(), [f, p] = y(a ? [...lx, ...l] : l), u = l.find((x) => x.type === "search"), j = s ? /* @__PURE__ */ e.jsx(S, {}) : /* @__PURE__ */ e.jsx("div", {});
   return /* @__PURE__ */ e.jsx(
     B,
     {
diff --git a/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/index.js b/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/index.js
index ed73945..2e33153 100644
--- a/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/index.js
+++ b/node_modules/@swmansion/t-rex-ui/dist/components/Navbar/index.js
@@ -1,12 +1,14 @@
 import { j as o } from "../../jsx-runtime-Bfani28R.js";
 import u from "./Layout/index.js";
 import f from "./Content/index.js";
+
 function j({
   heroImages: a,
   titleImages: m,
   isAlgoliaActive: r = !0,
   isThemeSwitcherShown: t = !0,
-  useLandingLogoDualVariant: n = !1
+  useLandingLogoDualVariant: n = !1,
+  landingItems: l = []
 }) {
   return /* @__PURE__ */ o.jsx(
     u,
@@ -20,7 +22,8 @@ function j({
           isAlgoliaActive: r,
           useLandingLogoDualVariant: n,
           heroImages: a,
-          titleImages: m
+          titleImages: m,
+          landingItems: l,
         }
       )
     }
